tensorflow/core/kernels/tile_op_test.cc

```diff
@@ -0,0 +1,158 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```

Comment:
C++ unit tests are for things that can't easily be tested in Python (at least in my opinion).